<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
    <item>
     <widget class="QLabel" name="mainTitle">
      <property name="styleSheet">
       <string>font-size:28px; font-weight:bold; color:#2d415a; margin:20px 0 30px 0;</string>
      </property>
      <property name="text">
       <string>MMNET</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_step1">
      <property name="title">
       <string>第一步：上传文件</string>
      </property>
      <layout class="QHBoxLayout" name="hLayout1">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>QPushButton{background-color:#4f8cff;color:white;font-size:16px;border-radius:8px;} QPushButton:hover{background-color:#357ae8;}</string>
         </property>
         <property name="text">
          <string>上传基因文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>QPushButton{background-color:#4f8cff;color:white;font-size:16px;border-radius:8px;} QPushButton:hover{background-color:#357ae8;}</string>
         </property>
         <property name="text">
          <string>上传表型文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>QPushButton{background-color:#f7b32b;color:white;font-size:16px;border-radius:8px;} QPushButton:hover{background-color:#e09e1b;}</string>
         </property>
         <property name="text">
          <string>上传待预测文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_step2">
      <property name="title">
       <string>第二步：运行MMNET</string>
      </property>
      <layout class="QVBoxLayout" name="vLayout2">
       <item>
        <layout class="QHBoxLayout" name="hLayout2">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Run Model</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QPushButton{background-color:#43b97f;color:white;font-size:16px;border-radius:8px;} QPushButton:hover{background-color:#2e8b57;}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_step2">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="format">
          <string>训练进度：%p%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>QProgressBar {
    height: 24px;
    font-size: 14px;
    border: 1px solid #cccccc;
    border-radius: 8px;
    background: #f5f5f5;
}
QProgressBar::chunk {
    background-color: #43b97f;
    border-radius: 8px;
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_step3">
      <property name="title">
       <string>第三步：预测</string>
      </property>
      <layout class="QVBoxLayout" name="vLayout3">
       <item>
        <layout class="QHBoxLayout" name="hLayout3">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QPushButton{background-color:#f7b32b;color:white;font-size:16px;border-radius:8px;} QPushButton:hover{background-color:#e09e1b;}</string>
           </property>
           <property name="text">
            <string>开始预测</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_download_pred">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QPushButton{background-color:#43b97f;color:white;font-size:16px;border-radius:8px;}
QPushButton:hover{background-color:#2e8b57;}
QPushButton:disabled{background-color:#cccccc;color:#888888;border:1px solid #bbbbbb;}</string>
           </property>
           <property name="text">
            <string>下载结果文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_predict_status">
         <property name="text">
          <string></string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>font-size:14px;color:#2d415a;margin-top:8px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
</ui>
